src/BenchmarkDotNet/Toolchains/Toolchain.cs

```diff
@@ -64,7 +64,7 @@ internal static bool IsCliPathInvalid(string customDotNetCliPath, BenchmarkCase
             if (string.IsNullOrEmpty(customDotNetCliPath) && !HostEnvironmentInfo.GetCurrent().IsDotNetCliInstalled())
             {
                 validationError = new ValidationError(true,
-                    $"BenchmarkDotNet requires dotnet cli to be installed or path to local dotnet cli provided in explicit way using `--cli` argument, benchmark '{benchmarkCase.DisplayInfo}' will not be executed",
+                    $"BenchmarkDotNet requires dotnet sdk to be installed or path to local dotnet cli provided in explicit way using `--cli` argument, benchmark '{benchmarkCase.DisplayInfo}' will not be executed",
```

Comment:
nit: I would uppercase the SDK

```suggestion
                    $"BenchmarkDotNet requires dotnet SDK to be installed or path to local dotnet cli provided in explicit way using `--cli` argument, benchmark '{benchmarkCase.DisplayInfo}' will not be executed",
```